.rhcicd/pr_check.sh

```diff
@@ -36,8 +36,27 @@ CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
 curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
 
 # Run ui tests
+export EXTRA_DEPLOY_ARGS="image-builder-crc"
```

Comment:
@ezr-ondrej This comment will start a larger conversation probably appropriate for slack, but I see this being annoying to maintain weekly with QE having to either assume an image is the right one in ephemeral, or constantly keep this UUID in sync.

How about injecting this via an init container in the clowdapp template, that only executes in ephemeral?

That way its not only part of this PR check, but any ephemeral deployment of provisioning has an image available to deploy.